interp/constrextern.ml

```diff
@@ -29,6 +29,7 @@ open Pattern
 open Detyping
 
 module NamedDecl = Context.Named.Declaration
+module CS = Structures
```

Comment:
`CS` looks like a remainder of the temporary commit when it was called `CanonicalStructure`!

Reply:
Yes, I should go back to open I guess.